### PR TITLE
Website: Add Rating control

### PR DESCRIPTION
**Pull request by cschleiden**
#### Description of changes

Add `Ratings` control to website

### CODE REVIEW

**Comment by cschleiden** [comment]
@dzearing @mikewheaton is this the way to update the website?

**Comment by mikewheaton** [comment]
You can change directory to `apps/fabric-website` and then run `npm start` to get a local build of the website. You should see your Ratings control at http://localhost:4321/index.html#/components/ratings

**Comment by cschleiden** [comment]
@mikewheaton yep that worked, just wanted to make sure there is nothing else to change.

File: apps/fabric-website/src/pages/Components/RatingComponentPage.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+export class RatingComponentPage extends React.Component<any, any> {
+  public render() {
+    return (
+      <div ref='pageElement'>
+        <ComponentPage>
+          <PageHeader pageTitle='Ratings' backgroundColor='#038387'
+            links={
+              [
+                {
+                  'text': 'Overview',
```
Comment (mikewheaton) [inline apps/fabric-website/src/pages/Components/RatingComponentPage.tsx]: Do we have any design guidance for this component?

**Comment by micahgodbolt** [comment]
@cschleiden can you drop a screenshot of the new component page so we can see how this renders?

**Comment by micahgodbolt** [comment]
Not getting any test image results. When I do --debug, I see the following errors in console.

image

**Comment by cschleiden** [comment]
Yeah, something is weird, didn't have latest master and after merging it doesn't show up. I'll take another look

**Comment by cschleiden** [comment]
@micahgodbolt had to rebuild locally, but it seems to be working fine. It renders like this:

image

**Pull request merged**